src/Common/NamedCollections/NamedCollections.cpp

```diff
@@ -223,6 +223,16 @@ class NamedCollection::Impl
             keys.insert(key);
     }
 
+    bool isOverridable(const Key & key, const bool default_value)
+    {
+        const auto overridable = Configuration::isOverridable(*config, key);
```

Comment:
It seems we use `Configuration::isOverridable` only here? Then may be to simplify it is better this way instead of adding additional function
```
const auto overridable = Configuration::getConfigValueOrDefault<bool>(*config, key + "[@overridable]", default_value);
```